definitions/npm/react-redux_v5.x.x/flow_v0.62.0-/test_connect.js

```diff
@@ -39,6 +39,100 @@ function testPassingPropsToConnectedComponent() {
   connect(mapStateToProps)('');
 }
 
+function doesNotRequireDefinedComponentToTypeCheck1case() {
```

Comment:
I could restructure the tests soonish, but I rather do it in separate PR to keep current changes clear.